test/stub/keyring-bridge.js

```diff
@@ -173,8 +173,14 @@ export class FakeLedgerBridge extends FakeKeyringBridge {
     //   common,
     // }).sign(Buffer.from(KNOWN_PRIVATE_KEYS[0], 'hex'));
 
+    //removing r, s, v values from the unsigned tx
```

Comment:
Nit: Maybe worth explaining why we're doing this. e.g. that Ledger uses `v` to communicate the chain ID, but we're removing it because these values are not a valid signature at this point.

Also, thoughts on setting `chainId` based on `v` rather than hard-coding it? Maybe not necessary, I can't think of a compelling example where we'd want a non-1337 hardware wallet e2e test right now, but I am not sure.

Reply:
thank you Mark, I've added a more clarifying comment :pray: